easybuild/easyconfigs/m/MView/MView-1.49-goolf-1.4.10-Perl-5.16.3.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = "Tarball"
+
+name = 'MView'
+version = '1.49'
+versionsuffix = '-Perl-5.16.3'
+
+homepage = 'http://sourceforge.net/projects/bio-mview/'
+description = """ MView reformats the results of a sequence database search or a 
+ multiple alignment, optionally adding HTML markup."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = [('http://sourceforge.net/projects/bio-mview/files/bio-mview/mview-%(version)s/', 'download')]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('Perl', '5.16.3')]
```

Comment:
maybe define the perl version once and use it both here and in the `versionsuffix`?